.github/workflows/test.yml

```
@@ -40,16 +40,30 @@ jobs:
     if: github.repository_owner == 'getsentry'
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         os: [ubuntu-24.04, ubuntu-24.04-arm]
-    name: ${{ matrix.os == 'ubuntu-24.04-arm' && 'integration test (arm64)' || 'integration test' }}
+        container_engine: ['docker'] # TODO: add 'podman' into the list
+    name: ${{ matrix.os == 'ubuntu-24.04-arm' && (matrix.container_engine == 'docker' && 'integration test (arm64)' || 'integration test (arm64 podman)') || (matrix.container_engine == 'docker' && 'integration test' || 'integration test (podman)') }}
```

I hate this so much 🤣🤣🤣

---

I don't want to change the configuration for the required checks, therefore I'm keeping it to be "integration test" for docker + amd64. Otherwise, we can go with the formatting. So.....

```suggestion
    name: ${{ (matrix.os == 'ubuntu-24.04-arm' && matrix.container_engine == 'docker') && 'integration test' || format("integration test ({0}) ({1})", matrix.container_engine, ${{ matrix.os == 'ubuntu-24.04-arm' && "arm64" || "amd64" }}) }}
```

I don't think this would work